lld/COFF/Config.h

```diff
@@ -24,9 +24,9 @@
 
 namespace lld::coff {
 
+using llvm::StringRef;
 using llvm::COFF::IMAGE_FILE_MACHINE_UNKNOWN;
 using llvm::COFF::WindowsSubsystem;
-using llvm::StringRef;
```

Comment:
Wait, this is only clang-format right?  This doesn't look like a formatting change at all.

Reply:
This appears to be: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#sortusingdeclarations

Reply:
and I would say sorting things where semantics don't change alphabetically (or rather by "ASCII") does more good than harm...

Reply:
Didn't mean to propose a change to "ASCII" here, mainly to illustrate what happened. It's probably best to avoid option changes mostly.